exercises/darts/darts_test.py

```diff
@@ -30,20 +31,20 @@ def test_just_outside_the_inner_circle(self):
         self.assertEqual(score(0.8, -0.8), 5)
 
     def test_just_within_the_middle_circle(self):
-        self.assertAlmostEqual(score(-3.5, 3.5), 5)
+        self.assertEqual(score(-3.5, 3.5), 5)
```

Comment:
Interesting that the original used `assertAlmostEqual` for floats with very few decimal places. Still, it may be useful to keep this.

@yawpitch, thoughts?

Reply:
Not sure why we’d ever need assertAlmostEqual... **score** always returns an int, or if for some reason the student decides to at most a float that’s equivalent to an int.